### PR TITLE
changed button, fixed load issues with profile pictures

### PR DESCRIPTION
**Pull request by Kennie12345**

### CODE REVIEW

File: examples/student.assistance.chat/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "@headlessui/react": "^1.7.10",
     "@heroicons/react": "^2.0.14",
     "@next/font": "13.1.6",
+    "@next/swc-darwin-x64": "^13.1.6",
```
Comment (SimonBiggs) [inline examples/student.assistance.chat/package.json]: @Kennie12345 this package makes it so that it can't build on the server...

**Review by SimonBiggs — CHANGES_REQUESTED** [review]
@Kennie12345, there's a package issue. See comment.

**Review by Kennie12345 — COMMENTED** [review]
fix for package issue

**Comment by SimonBiggs** [comment]
@Kennie12345, I'm going to close this PR.

Feel free to re-submit it if you'd like these changes merged.

**Comment by Kennie12345** [comment]
Thanks bro!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**